doc/install/get-packages.rst

```diff
@@ -114,9 +90,8 @@ bugfixes are backported to LTS releases until their retirement. Since retired
 releases are no longer maintained, we recommend that users upgrade their
 clusters regularly - preferably to the latest LTS release.
 
-The most recent LTS release is Jewel (10.2.x).
-
-.. tip:: For international users: There might be a mirror close to you where download Ceph from. For more information see: `Ceph Mirrors`_.
+.. tip:: For non-US users: There might be a mirror close to you where
+         download Ceph from. For more information see: `Ceph Mirrors`_.
```

Comment:
This sentence was previously broken, but since you're touching it you can fix it. :) "where you can download"

(It reoccurs below a couple times as well.)